code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -282,12 +282,15 @@
 	new /obj/item/clothing/head/helmet/alt(src)
 	new /obj/item/clothing/mask/gas/sechailer(src)
 	new /obj/item/clothing/suit/armor/bulletproof(src)
+
 /obj/structure/closet/secure_closet/lethalshots
 	name = "shotgun lethal rounds"
```

Comment:
If you are adding electrostaffs in here, perhaps you should also rename this closet.